examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt

```diff
@@ -113,7 +114,10 @@ class KotlinExampleActivity : Activity() {
         showStatus("\nPerforming basic Query operation...")
         showStatus("Number of persons: ${realm.where<Person>().count()}")
 
-        val results = realm.where<Person>().equalTo(Person::age, 99).findAll()
+        val ageCriteria = 99
+        val results = realm.where<Person>()
+                .equalTo(Person::age.name, ageCriteria)
```

Comment:
Do we want to do this in our example when this approach doesn't work for linked fields either? Or should we just move back to "age"?

Reply:
hmm.. I liked the idea of illustrating that Kotlin gives you the ability to do this, making a safe reference to the property.  I can change it to use a string though if you feel strongly.

Reply:
I agree it is nice that it works, but since these examples are in many cases copy-pasted, having code in here that doesn't work in all cases is probably a bad idea. We could perhaps just add a comment about it, and just quickly describe advantages and pitfalls of using it, but keep using Strings in actual code?

Reply:
So this would work even with proguard enabled? That is nice!

Reply:
Yeah, I will change it to strings with a comment about what's possible in kotlin for simple field access.

I also think we may want to point people to `realmfieldnameshelper`.  Maybe I can add that in the comment too?  Any objection to that?

Reply:
No, thats fine.